sdks/urbackend-python/README.md

```
@@ -0,0 +1,328 @@
+| `me(token?)` | — | `dict` (profile) |
+| `logout(token?)` | — | `dict` |
+| `refresh_token(refresh_token?)` | — | `dict` (new session) |
+| `update_profile(payload, token?)` | — | `dict` |
+| `change_password(current, new, token?)` | — | `dict` |
+| `public_profile(username)` | — | `dict` |
+| `verify_email(email, otp)` | — | `dict` |
+| `resend_verification_otp(email)` | — | `dict` |
+| `request_password_reset(email)` | — | `dict` |
+| `reset_password(email, otp, new_password)` | — | `dict` |
+| `social_start_url(provider)` | `"github"` \| `"google"` | `str` |
+| `social_exchange(rt_code, token)` | — | `dict` || `set_token(token)` | — | `None` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)